app/src/main/java/com/github/epfl/meili/posts/PostActivity.kt

```diff
@@ -58,10 +58,6 @@ class PostActivity : AppCompatActivity(), ClickListener, ListSorter<Comment> {
         initRecyclerAdapter()
         initLoggedInListener()
         initSorting(findViewById(R.id.sort_spinner), R.array.sort_array_comments)
-
-        findViewById<TextView>(R.id.userName).setOnClickListener {
```

Comment:
Why do you delete this? So we no longer allow user to check the profile from post?

Reply:
No, it was redundant, this is already done in the MeiliRecyclerViewAdapter, but it would be nice if you could double check

Reply:
For me it is not working, when I'm on a post and click on the name or photo of the sender, it does not go to the profile

Reply:
Good catch, I have fixed it.